content/docs/command-reference/import.md

```diff
@@ -104,15 +104,18 @@ To actually [version the data](/doc/start/data-and-model-versioning), `git add`
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
-- `--no-exec` - create the import `.dvc` file but don't download `url` (assumes
+- `--no-exec` - create the import `.dvc` file without accessing `url` (assumes
   that the data source is valid). This is useful if you need to define the
   project imports quickly, and import the data later (use `dvc update` to finish
   the operation(s)).
 
-- `--no-download` - create the import `.dvc` with data checksums but without
-  downloading the associated data. This is useful if you need track changes in
-  remote data but do not (yet) need to download data to the local workspace.
-  Data can be later downloaded using `dvc pull`.
+- `--no-download` - create the import `.dvc` file including the
+  [version](https://git-scm.com/docs/revisions) for the external dependency but
+  without downloading the associated data. This is useful if you need track
+  changes in remote data without using local storage space (yet). The data can
+  be downloaded later using `dvc pull`, but this will fail if the `url` no
+  longer matches the hash values. File hashes can be updated using
```

Comment:
Is it possible that `dvc pull` on an `import` will fail? Since it relies on a Git hash, seems like this shouldn't be an issue for `import`.

Reply:
Good point. Well, the git hash might be missing, but I think we can safely remove "but this will fail..."

Reply:
Removed the phrase. Note that if the hash does exist, dvc will print:
> ERROR: failed to pull data from the cloud - unknown Git revision 'badhash'